docs/paywall2.md

```diff
@@ -241,16 +241,17 @@ Usage example:
 
 ## Standalone Functionalities
 
-The InPlayer Paywall supports standalone functionalities for several purposes: login/logout button functionality, as well as accessing the 'my account' section of the paywall application.
+The InPlayer paywall also supports standalone functionalities involving the login/logout button functionality, as well as accessing the ‘my account’ section of the paywall application.
```

Comment:
Beware when you copy and paste text from Word. It can change the formatting of some special characters like the quotes in this case.

Reply:
Thank you, I'll be sure to pay more attention from now on.